### DISCUSSION TITLE
MacOS  下内存占用问题

### DISCUSSION THREAD

**Discussion by le0zh** [CefView/QCefView#177] [General]
在 Mac 下跑 QCefViewTest 示例工程（release NCW 模式），主进程内存 200MB+，相比 windows 上要大许多，不太清楚是什么导致。

<img width="609" alt="image" src="https://user-images.githubusercontent.com/3633972/201332908-08e78a42-722c-455e-96cb-f8e6f1ba2f94.png">

**Comment by wanding0101005** [comment]
dyld: Library not loaded: @rpath/QtGui.framework/Versions/5/QtGui
  Referenced from: /Users/wanding/Desktop/QCefView-1.0-7/.build/macos.x86_64/example/QCefViewTest/output/Debug/bin/QCefViewTest.app/Contents/Frameworks/QtWidgets.framework/Versions/5/QtWidgets_debug

为什么我用xcode编译的时候抱这个错误呀，请问您怎么编译过的呀
